lib/generators/rolemodel/webpack/templates/app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,5 @@
+// This is a manifest file that'll be compiled into application.css, which will include
```

Comment:
So this settles that for an upgrade we'll move all our stylesheets to `app/assets` and keep the javascript in `app/javascript` correct? Do we like that pattern?